brainiak/matnorm_models/noise_covs.py

```diff
@@ -0,0 +1,268 @@
+import tensorflow as tf
+import numpy as np
+import abc
+from .helpers import define_scope, xx_t
+
+
+class NoiseCovBase:
```

Comment:
I think that the cholesky factor should always be a property of these covariances. For your consideration...